tests/federation/transport/test_server.py

```diff
@@ -38,3 +38,19 @@ def test_open_public_room_list_over_federation(self):
             "/_matrix/federation/v1/publicRooms",
         )
         self.assertEqual(200, channel.code)
+
+    @DEBUG
+    def test_edu_debugging_doesnt_explode(self):
+        """Sanity check fed. RX succeeds with `synapse.debug_8631` logging enabled.
```

Comment:
I don't know what "RX" means, could you update this comment please?

Reply:
TX: transmit
RX: receive

not sure where the abbreviation comes from. Synapse uses the former in `transaction_manager.py` and `per_destination_queue.py`. Slightly surprised to see it doesn't use the latter.

Reply:
Possible etymology: https://en.wikipedia.org/wiki/RS-232

Reply:
Thanks! fwiw I just read `TX` in those file as an abbreviated version of `txn` for "transaction", TIL TX and RX mean transmit and receive.